### PR TITLE
[UG] Show and Sort Command

### PR DESCRIPTION
**Pull request by jiaweiteo**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -251,6 +262,74 @@ Format: `edit -friend INDEX n/NAME m/MOBILE_NUMBER p/PASSPORT_NUMBER​`
...
+
+Switches the current Ui view to show Activity tab under the travel plan object panel.
+This command has no effect in a wishlist as a wishlist only contains activities.
+
+Format/ Example: `show -activity`
+
+### 2. Showing Accommodation Tab
+
+Switches the current Ui view to show Accommodation tab under the travel plan object panel.
+This command has no effect in a wishlist as a wishlist does not contain Accommodation.
```
Comment (lyeyixian) [inline docs/UserGuide.md:277]: Do we allow them to type this command when in wishlist? Or do we output error message?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -251,6 +262,74 @@ Format: `edit -friend INDEX n/NAME m/MOBILE_NUMBER p/PASSPORT_NUMBER​`
 
 Example: `edit -friend 3 n/John m/91234567 p/E7654321`
 
+##Show
+
+### 1. Showing Activity Tab
```
Comment (lyeyixian) [inline docs/UserGuide.md:267]: I think you miss out the (L) / (G)?

**Review by lyeyixian — APPROVED** [review]
LGTM but you need to merge according to the timeline cos your branch is all linked together i think. If you merged this, the previous changes will also be merged without the new changes

File: docs/UserGuide.md (hunk 1 above)
Comment (jiaweiteo) [reply docs/UserGuide.md:277]: Error msg thrown

File: docs/UserGuide.md (hunk 2 above)
Comment (jiaweiteo) [reply docs/UserGuide.md:267]: ok

**Pull request merged**